CHANGELOG.md

```diff
@@ -34,6 +33,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Fixed: Some seeds being considered impossible when finding a progressive item in an area where a later item in the progressive chain is required to leave.
 
+### Metroid Prime 2: Echoes
+
+- Fixed: Cannon Ball and all multiplayer pickups will never be given as a starting item, as they cannot due to patcher limitations.
```

Comment:
I wouldve just copypasted what dunc wrote here haha.
Yours sounds a little bit weird grammatically and slightly less clear